internal/learn/iota-token/staking-iota.md

```diff
@@ -0,0 +1,100 @@
+---
+id: staking-iota
+title: Staking IOTA
+description: Everything you need to know about all the different upcoming staking opportunities in IOTA, Assembly, and Shimmer network.
+---
+
+# Staking in the IOTA network ecosystem
+
+IOTA introduced two new networks that will exist next to the main network: the Shimmer staging network and the Assembly trustless smart contract network. This marks a historical event and a vast increase of utility for the IOTA token.
+Both networks will launch with their own native tokens: SMR for Shimmer and ASMB for Assembly. With the new tokenomics and incentive structure around these networks, it is now possible to stake tokens.
+
+- Find the blog post introducing IOTA staking [here](https://blog.iota.org/introducing-iota-staking/).
+- Find the announcement of the start of IOTA staking [here](https://blog.iota.org/iota-staking-start/).
```

Comment:
I think it is visually more clear if we wrap this in a note or info admonition.

Reply:
```suggestion
:::info

- Find the blog post introducing IOTA staking [here](https://blog.iota.org/introducing-iota-staking/).
- Find the announcement of the start of IOTA staking [here](https://blog.iota.org/iota-staking-start/).

:::
```